src/Repositories/MigrationRepository.php

```diff
@@ -50,7 +50,7 @@ public function delete(string $fileName): bool
 
     public function truncate(): void
```

Comment:
This naming is now misleading. Shall we create a new method instead (named `deleteAll` or `purge`) and mark `truncate` as deprecated?

Reply:
Yea I think that's a good idea 👍 , which method name would you prefer?

Reply:
Added `deleteAll()` and deprecated `truncate()`. Let me know if you have any further requests! 👍